doctrine/doctrine-migrations-bundle/1.2/etc/packages/migrations.yaml

```diff
@@ -0,0 +1,2 @@
+doctrine_migrations:
+    dir_name: '%kernel.project_dir%/DoctrineMigrations'
```

Comment:
@fabpot Should `src/migrations` not be `src/Migrations`. Because the Entity folder has a capital letter to in the directory. Let me know, I've added it with a Capital letter for now.